frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  const error = MultipleErrorWrapper(
+    `Unable to load approval page`,
+    {
+      isApprovalsError,
+    },
+    ErrorWrapper
+  )
+  if (error) return error
+
+  const getUploadCategory = (approvalCategory: ApprovalCategory) =>
+    approvalCategory === 'Upload' ? 'models' : 'deployments'
+
+  if (isApprovalsLoading || !approvals) {
+    return <Paper sx={{ mt: 2, mb: 2 }} />
+  }
+  return (
+    <Paper sx={{ mt: 2, mb: 2, pb: 2 }}>
+      <Typography sx={{ p: 3 }} variant='h4'>
+        {category === 'Upload' ? 'Models' : 'Deployments'}
+      </Typography>
+      {approvals.map((approval: Approval) => (
+        <ApprovalItem
+          approval={approval}
+          category={approval.approvalCategory}
+          version={approval.version as VersionDoc}
+          key={approval._id}
+          mutateApprovals={mutateApprovals}
+        />
+      ))}
+
+      {approvals.length === 0 && (
+        <EmptyBlob text={`All done! No ${getUploadCategory(category)} are waiting for approval.`} />
+      )}
+    </Paper>
+  )
+}
+
+type ApprovalItemProps = {
+  approval: Approval // changing this from 'any' to 'Approval' causes errors
+  category: string
+  version: VersionDoc
+
+  mutateApprovals: () => void
```

Comment:
We shouldn't be passing this in via props. As it stands, this component would break if I passed something like `mutateApprovals={() => undefined}` to it, but that adheres to our props interface. So we should instead access `mutateApprovals` again via the same hook as before. I've updated the props to allow for this. I've also removed `version` from your props interface as it wasn't being used:
```
type ApprovalItemProps = {
  approval: Approval 
  approvalCategory: ApprovalCategory
  filter: ApprovalFilterType
}

function ApprovalItem({ approval, approvalCategory, filter }: ApprovalItemProps) {
// There's a bunch of hooks here

// Then add the following line next to the useGetNumApprovals hook 
const { mutateApprovals } = useListApprovals(category, filter)
const { mutateNumApprovals } = useGetNumApprovals()
```